package.json

```diff
@@ -29,7 +29,7 @@
     "@babel/runtime": "^7.6.3",
     "bluebird": "^3.5.4",
     "browserify": "^16.2.3",
-    "lev-react-components": "^0.8.0",
+    "lev-react-components": "^0.8.1",
```

Comment:
^0.9.0

Reply:
This should also fix your PR tests I think, once version `0.9.0` exists